### PR TITLE
issue132 - text updated for considering DTLS implementation

### PR DESCRIPTION
**Pull request by teiclap**
#issue132 - Updated with considerations towards record size limits due to DTLS implementation

### CODE REVIEW

**Comment by teiclap** [comment]
Text has been aligned with #130, the relationship between DTLS maximum record size and rfc8449 has been clarified.

**Review by gloinul — COMMENTED** [review]
This text appears to be duplicating the change in #167?

**Comment by gloinul** [comment]
Realise that this text is in the introduction section and the text is to provide an overview. Thus, the text should be reformulated to not use RFC 2119 language and instead rather be informative that implementation that support RFC 8449 can negotiate reduced sized DTLS records that allows reducing buffering. I am uncertain if it might also provide improved interoperability with any implementation that has limited sized DTLS records?

**Comment by gloinul** [comment]
With the introduction changes that are overlapping with this PR in #169. I would suggest that the normative text part is moved into that PR and this abandoned to avoid having to deal with overlapping text changes.

File: draft-ietf-tsvwg-dtls-over-sctp-bis.md [hunk 1]
```diff
@@ -481,6 +486,12 @@ connection can be terminated and the associated keying material discarded.
    processing. Making it the ULP responsible for handling any resource
    contention related to large user messages.
 
+   It is mandated that DTLS supports the maximum record size of 2<sup>14</sup> bytes.
```
Comment (emanjon) [inline draft-ietf-tsvwg-dtls-over-sctp-bis.md:489]: Martin wrote:

"you might find that things fail if you **handshake** with 214 bytes in a record."

Should the requirement be general or should it only be for application data?

**Comment by emanjon** [comment]
Seems to be very much overlap with #167

File: draft-ietf-tsvwg-dtls-over-sctp-bis.md (hunk 1 above)
Comment (gloinul) [reply draft-ietf-tsvwg-dtls-over-sctp-bis.md:489]: It was intended for application data, not to apply to other DTLS messages like handshake etc.

**Comment by teiclap** [comment]
Resolved the conflict, ready for merging.

**Review by gloinul — APPROVED** [review]

**Pull request merged**